bower.json

```diff
@@ -22,7 +22,7 @@
     "angular-bootstrap-switch": "~0.5.1",
     "angular-dragdrop": "~1.0.13",
     "angular-mocks": "~1.5.8",
-    "angular-patternfly-sass": "~3.15.0",
+    "angular-patternfly-sass": "~3.21.0",
```

Comment:
are you missing 
`"angular-drag-and-drop-lists": "2.0.0",`
line ?
I can't find it , but I do not know if we need it :-) see:
https://github.com/patternfly/angular-patternfly/pull/422/files